go/src/lib/lomipc/client_transport.go

```diff
@@ -46,6 +46,10 @@ var ClientCall = txCallClient
  *  Appropriate error object on failure
  */
 func (tx *ClientTx) RegisterClient(client string) error {
+    if tx.clientRpc != nil {
+        /* This context is local to client. So this bug will not be hit on restart */
+        return LogError("Already registered (%s). De-register & re-register", tx.clientName)
```

Comment:
caller has to parse message differentiate b/w this and other errors . Is that expected ?

Reply:
No. Log errors are for human read only. This should not happen in code. So if failure happens, this is more for debugging & fix. This goes for all log error messages.